### PR TITLE
Admin

### PR DESCRIPTION
**Pull request by mashg**

### CODE REVIEW

**Review by MJiason — APPROVED** [review]

File: src/app/store/service/employee/employeeService.ts [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+//   const body = {
+//     email: editEmployeeData.email,
+//     status: editEmployeeData.status,
+//     firstName: editEmployeeData.firstName,
+//     lastName: editEmployeeData.lastName,
+//     userRole: editEmployeeData.userRole,
+//     phoneNumber: editEmployeeData.phoneNumber,
+//   };
+//   return this.http.post(url, body);
+// }
```
Comment (Dima2057) [inline src/app/store/service/employee/employeeService.ts:69]: Please, remove this stub. Store it in local version.

File: src/app/store/service/employee/employeeService.ts [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+@Injectable({
+  providedIn: 'root',
+})
+export class EmployeeService {
+  private hostURL = 'https://spacey-backend.herokuapp.com';
+  private employeeUrl = `${this.hostURL}/employees`;
+  private searchEmployeeUrl = this.employeeUrl + '/search';
+  private addEmployeeUrl = `${this.employeeUrl}/create`;
+
+  // private editEmployeeUrl = this.employeeUrl + '/update';
```
Comment (Dima2057) [inline src/app/store/service/employee/employeeService.ts:22]: Please, remove this stub. Store it in local version.

File: src/app/components/admin-manage/admin-employee.component.ts [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    'lastName',
+    'userRole',
+    'phoneNumber',
+    'email',
+    'status',
+    'actions',
+  ];
+  dataSource = new MatTableDataSource<EmployeeModel>();
+  employees!: EmployeeModel[];
+  // private searchText$ = new Subject<string>();
```
Comment (Dima2057) [inline src/app/components/admin-manage/admin-employee.component.ts:27]: Please, remove this stub. Store it in local version.

File: src/app/components/admin-edit/admin-edit.component.ts [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+  //       this.editEmployeeForm.controls.email.enable();
+  //       this.editEmployeeForm.controls.firstName.enable();
+  //       this.editEmployeeForm.controls.lastName.enable();
+  //       this.editEmployeeForm.controls.userRole.enable();
+  //       this.editEmployeeForm.controls.status.enable();
+  //       this.editEmployeeForm.controls.phoneNumber.enable();
+  //     }
+  //   );
+  //   }
+}
```
Comment (Dima2057) [inline src/app/components/admin-edit/admin-edit.component.ts:90]: Please, remove this stub. Store it in local version.

File: src/app/components/admin-edit/admin-edit.component.html [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+<!--      [disabled]="!editEmployeeForm.valid"-->
+<!--      disabled-->
+<!--      type="submit"-->
+<!--      color="accent"-->
+<!--    >-->
+<!--      EDIT-->
+<!--    </button>-->
+<!--  </div>-->
+<!--  <span></span>-->
+<!--</form>-->
```
Comment (Dima2057) [inline src/app/components/admin-edit/admin-edit.component.html:117]: Please, remove this stub. Store it in local version.

File: src/app/components/admin-add/admin-add.component.html [hunk 6]
```diff
@@ -0,0 +1,124 @@
...
+        <!--          placeholder="Status"-->
+
+        <mat-select>
+          <mat-option value="option1">Active</mat-option>
+          <mat-option value="option2">Terminated</mat-option>
+          <mat-option value="option3">Inactive</mat-option>
+        </mat-select>
+        <!--        <mat-error *ngIf="addEmployeeForm.get('status')?.hasError('required')">-->
+        <!--          Status is required!-->
+        <!--        </mat-error>-->
```
Comment (Dima2057) [inline src/app/components/admin-add/admin-add.component.html:107]: Please, remove this stub. Store it in local version.

**Review by Dima2057 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**